SECURITY.md

```diff
@@ -0,0 +1,93 @@
+# Notation Security Process and Policy
+This document provides the details on the Notation security policy and details the process surrounding security handling including a how to report a security vulnerability for anything within the Notary Project organization. 
```

Comment:
nit:
extra `a` before `how to`

Reply:
I believe this is grammatically correct - the `a` relates to the `report` noun.